tests/sentry/logging/test_audit.py

```diff
@@ -1,73 +0,0 @@
-from __future__ import absolute_import
```

Comment:
Should repurpose these tests for `TuringHandler` since they're still applicable.